### PR TITLE
Handle errors while getting external IP gracefully

### LINKED ISSUE #975: AWS error whitelisting external IP
In rare cases supergiant receives an error

```
failed to whitelist supergiant IP in master security group: InvalidParameterValue: CIDR block /32 is malformed\n\tstatus code: 400
```

This is caused by error in figuring out external ip address and passing this error silently 

https://github.com/supergiant/control/blob/master/pkg/util/util.go#L143
https://github.com/supergiant/control/blob/master/pkg/util/util.go#L148

### PR DESCRIPTION
**Pull request by stgleb**
Closes #975

### CODE REVIEW

File: pkg/workflows/steps/amazon/util.go [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+				timeout = timeout * 2
+			}
+		}
+	}
+
+	return publicIP, err
+}
+
+func findOutBoundIP() (string, error) {
+	serviceURL := "http://myexternalip.com/raw"
```
Comment (saheienko) [inline pkg/workflows/steps/amazon/util.go]: In case of `http://myexternalip.com/raw` outage cluster bootstrapping would be failed?

Comment (stgleb) [reply pkg/workflows/steps/amazon/util.go]: @saheienko yes, do you know better ways to figure out real external IP addr in network with random topology?

Comment (TheKLARKEN) [reply pkg/workflows/steps/amazon/util.go]: You can curl the AWS IP:

`curl http://169.254.169.254/latest/meta-data/public-ipv4`
https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-instance-addressing.html

Comment (TheKLARKEN) [reply pkg/workflows/steps/amazon/util.go]: @stgleb ^ Would that work?

Comment (stgleb) [reply pkg/workflows/steps/amazon/util.go]: @TheKLARKEN this is for instance public IP address, but I need to get public IP of SG itself which
can be outside of AWS.

Comment (TheKLARKEN) [reply pkg/workflows/steps/amazon/util.go]: @stgleb Ahh, of SG itself. That makes sense. 🙂

**Pull request merged**